lib/services/user_service.dart

```diff
@@ -609,7 +609,7 @@ class UserService {
     }
   }
 
-  Future<void> setupTwoFactor(BuildContext context) async {
+  Future<void> setupTwoFactor(BuildContext context, Completer completer) async {
```

Comment:
Passing completer here seems weird. Let's discuss if we really need this.

Reply:
Will remove this in the future when the toggles are moved to a separate screen